### PR TITLE
ADEN-3731 Add Ubisoft tracking

### PR DESCRIPTION
**Pull request by v3rth**
## Links

* https://wikia-inc.atlassian.net/browse/ADEN-3731

### CODE REVIEW

File: server/app/views/_partials/tracking-ubisoft.hbs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!-- Begin Ubisoft Tag -->
+<script type="text/javascript">
+    (function () {
```
Comment (fraszczakszymon) [inline server/app/views/_partials/tracking-ubisoft.hbs]: Tabs please

File: server/app/views/_partials/tracking-ubisoft.hbs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            scriptElementWikia.src = url + "?rand=" + a;
+            document.body.appendChild(scriptElementWikia);
+        }
+
+        if (ubisoftTracking.enabled && ubisoftTracking.url) {
+            initializeUbisoftTracking(ubisoftTracking.url);
+        }
+    })();
+</script>
+<!-- End Ubisoft Tag -->
```
Comment (fraszczakszymon) [inline server/app/views/_partials/tracking-ubisoft.hbs]: New line please

**Comment by fraszczakszymon** [comment]
Minor notes and LGTM

**Comment by jacekjursza** [comment]
LGTM

File: server/app/views/_partials/tracking-ubisoft.hbs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<!-- Begin Ubisoft Tag -->
+<script type="text/javascript">
+	(function () {
+		var ubisoftTracking = M.prop('tracking.ubisoft'),
+            externalsEnabled = !M.prop('queryParams.noexternals');
```
Comment (fraszczakszymon) [inline server/app/views/_partials/tracking-ubisoft.hbs]: Tabs please :D

**Pull request merged**